src/redux/thunks/applyFile.ts

```diff
@@ -0,0 +1,78 @@
+import {spawn} from 'child_process';
+import log from 'loglevel';
+import {FileMapType} from '@models/appstate';
+import {setAlert} from '@redux/reducers/alert';
+import {AlertEnum, AlertType} from '@models/alert';
+import {AppDispatch} from '@redux/store';
+import {getShellPath} from '@utils/shell';
+import {setApplyingResource} from '@redux/reducers/main';
+import fs from 'fs';
+import {getAbsoluteFileEntryPath} from '@redux/services/fileEntry';
+
+/**
+ * Invokes kubectl for the content of the specified resource
+ */
+
+function applyFileToCluster(filePath: string, kubeconfig: string) {
```

Comment:
I think a better name for this would be `spawnProcessToApplyFile` or `spawnApplyFileProcess`